src/config.ts

```diff
@@ -18,3 +20,38 @@ export const nodeImportMapPath = join(root, "node.importmap");
 export const cache = join(root, ".cache");
 const map = existsSync(nodeImportMapPath) ? JSON.parse(readFileSync(nodeImportMapPath, { encoding: "utf8" })) : {};
 export const importmap = new ImportMap({ rootUrl: import.meta.url, map });
+
+interface OptionDefinition {
+  type?: string;
+  alias?: string;
+  default?: any;
+}
+
+interface Options {
+  args: string[];
+  options: Record<string, OptionDefinition>;
+}
+
+function parseArgs({ args, options: optionDefinitions }: Options) {
+  const parsedArgs: Record<string, string | boolean> = {};
+
+  for (let i = 0; i < args.length; i++) {
+    const arg = args[i].replace(/^-+/, "");
+    const optionDefinition = optionDefinitions[arg];
+
+    if (optionDefinition) {
+      const value = args[i + 1];
+      parsedArgs[optionDefinition.alias || arg] = value !== undefined ? value : optionDefinition.default || true;
+      i++;
+    }
+  }
+
+  return { values: parsedArgs };
+}
+
+const { values } = parseArgs({
+  args: argv.slice(2),
+  options: { "debug-node-loader": { alias: "d", type: "boolean", default: false } },
+});
+
+export const isDebuggingEnabled = (): boolean => values["debug-node-loader"] as boolean;
```

Comment:
`--debug` is standard, right?
I think we can [capture this differently](https://github.com/jspm/node-importmap-http-loader/pull/8/commits/de221cb343270afc848ccf5930d20f27d4815694#diff-39b2554fd18da165b59a6351b1aafff3714e2a80c1435f2de9706355b4d32351R196) as well.

Reply:
I wrote the explanation here
https://github.com/jspm/node-importmap-http-loader/pull/18#issuecomment-1710663018

And the reason for not using `--debug` is. Users can pass flags similar to this to their own apps. So, `--debug` is quite generic. If the main app uses it, it can clash with that

Reply:
And we prefer dashes vs camelcase?